From b37a944f55a5010bd08297a63db0275540922f32 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Thu, 22 Aug 2013 16:31:29 -0300
Subject: [PATCH] drm/vivante: Add ":00" sufix in returned bus Id

This makes the 3.0.35 compatible with a Xorg driver build for 3.5.7 or
newer kernels.

Upstream-Status: Inapropriate [embedded specific]

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 drivers/gpu/drm/vivante/vivante_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/vivante/vivante_drv.c b/drivers/gpu/drm/vivante/vivante_drv.c
index 4224608..cea360d 100644
--- a/drivers/gpu/drm/vivante/vivante_drv.c
+++ b/drivers/gpu/drm/vivante/vivante_drv.c
@@ -55,7 +55,7 @@
 
 #include "drm_pciids.h"
 
-static char platformdevicename[] = "Vivante GCCore";
+static char platformdevicename[] = "Vivante GCCore:00";
 static struct platform_device *pplatformdev;
 
 static struct drm_driver driver = {
-- 
1.8.4.rc1